### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Sites-Create Web Site_Keyboard: Focus is not visible on the "New Root", "Refresh", "Go Up", "OK" and "Cancel" buttons and toggle buttons while navigating through the Keyboard.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#126]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106

**MAS ID Violated**: MAS2.4.7

**Repro Steps:** 
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till "Web Sites" in the left menu section and Press Enter.
5. Tab till "Create Web Site" expandable button and Press Enter.
6. Verify that keyboard focus is visible on all the interactive controls while navigating through the keyboard.

**Actual Result:**
Focus is not visible on the "New Root", "Refresh", "Go Up", "OK" and "Cancel" buttons and toggle buttons while navigating through the Keyboard.

**Expected Result:**
When any interactive control receives keyboard focus, a rectangular border should appear around these controls so that user can differentiate clearly between the focused and non-focused controls on the page. 

**User Impact:**
This Success Criterion helps anyone who relies on the keyboard to operate the page, by letting them visually determine the component on which keyboard operations will interact at any point in time.
People with attention limitations, short term memory limitations, or limitations in executive processes benefit by being able to discover where the focus is located.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}

**Attachment:**
![bug 126_focus visibility issue 2](https://user-images.githubusercontent.com/38521133/44711161-57079d80-aacb-11e8-947c-a26e72dfa0d7.png)
![bug 126_focus visibility issue](https://user-images.githubusercontent.com/38521133/44711162-57a03400-aacb-11e8-9c4c-30a0ede553ca.png)

**Comment by jhkimnew** [comment]
Fixed

**Comment by mstechie** [comment]
GitHubTags:#A11y_Manage.IIS.Net;
